cg-spec/editors_draft.html

```diff
@@ -2542,6 +2557,16 @@ <h2>RDF-star Semantics</h2>
+
+      <p>The downside of this design is that IRIs in embedded triples don't co-denote like they do in the statement that the embedded triple represents. This can have unexpected effects and impairs the ability of embedded triples to reliably model common constructs like attributed relations in property graph style, reification or n-ary relations. It is common on the semantic web that data integration processes replace IRIs by co-denoting terms from other vocabularies, and this ability is indeed essential to the conception of the semantic web as a shared knowledge space without centralized vocabularies. Statements so modified still have the same meaning as only co-denoting terms are exchanged. Embedded triples however, although in most use cases supposed to present the hinge between a statement and its annotations, won't follow suit as they are fixed to their syntactic form. Annotations therefore may loose their connection with the statement they were meant to annotate.</p>
+
+      <p>This approach has been controversial in the Community Group and criticised as being rather unbalanced w.r.t. to actual use cases and indeed risky as it may be too surprising and unintuitive to be followed. Choosing opaque semantics as the default is substantiated by the monotonic nature of the RDF semantics, in which entailments once made can never be taken back by further semantic instructions. The RDF-star semantics consequently starts from an entailment-suppressing syntactic representation, arguing that it's always possible later to derive the entailments on embedded triples that the referentially opaque semantics initially suppresses, thereby re-instantiating e.g. co-denotation, sub-classing and other derived knowledge. The counter argument to this concept is that it is still quite unclear how to initiate such entailments in practice in an efficient and principled way. Proposals brought forward <a href="https://github.com/w3c/rdf-star/issues/170">so far</a> either introduce referential transparency in ad hoc ways per annotating property or annotated triple or rather sweepingly per graph or dataset. The former solution doesn't scale well in situations where the need for referential opacity is not the norm but the exception and isn't constrained to a small set f properties or triples, which is arguably the vast majority of operations on the semantic web. For this reason the latter solution would seem more sensible with per default referentially transparent semantics, switching to referential opacity on demand - per application or per graph or dataset. What is indeed not possible is to switch per property or statement from a default state of referential transparency to a more restricted opaque state as that would require the - non-monotonic and therefore unavailable - ability to take back entailments.</p>
```

Comment:
> The RDF-star semantics consequently starts from an entailment-suppressing syntactic representation

Repeating myself: [unless you provide me with a counter-example](https://github.com/w3c/rdf-star/pull/173#issuecomment-855436842), I insist that this assertion is plain wrong.

Reply:
I can't even find where to begin with this paragraph.

Reply:
> > The RDF-star semantics consequently starts from an entailment-suppressing syntactic representation
> 
> Repeating myself: [unless you provide me with a counter-example](https://github.com/w3c/rdf-star/pull/173#issuecomment-855436842), I insist that this assertion is plain wrong.

There you say "given any two RDF graphs G1 and G2, if G1 entails G2 in simple RDF entailment, then G1 entails G2 in the newly defined RDF-star entailment" but how would simple RDF entailment know what to do with embedded triples?

[Example 33](https://pr-preview.s3.amazonaws.com/w3c/rdf-star/185/524a524...rat10:5f77b06.html#ref-opacity-owl-entailment) is an example of how RDF-star starts from an entailment-suppressing syntactic representation, the whole point of the proposed semantics IIUC. What are we discussing about here? Did I use a wrong technical term? Did you even try to understand what I might want to say?

Reply:
@rat10 
> There you say "given any two RDF graphs G1 and G2, if G1 entails G2 in simple RDF entailment, then G1 entails G2 in the newly defined RDF-star entailment" but how would simple RDF entailment know what to do with embedded triples?

Indeed, when I write "RDF graph" (as opposed to "RDF-star graph"), I mean a graph that contains no embedded triple. The proposed semantics ensures that, for such graphs, the RDF-star entailment is the same as simple RDF entailment.

As for embedded triples, as you point out, RDF knows nothing about them. And therefore RDF semantics enforces no particular entailment about them. Those entailments that you argue are "suppressed" by the proposed RDF-star semantics were never there in the first place.